htmresearch/algorithms/column_pooler.py

```diff
@@ -53,7 +53,8 @@ def __init__(self,
                sampleSizeDistal=20,
                activationThresholdDistal=13,
                connectedPermanenceDistal=0.50,
-               distalSegmentInhibitionFactor=1.001,
+               distalSegmentInhibitionFactor=0.999,  #Should be <1
```

Comment:
Instead of putting "Should be <1" here, we should probably:

1. Update the docstring for distalSegmentInhibitionFactor below.
2. Change the `assert distalSegmentInhibitionFactor > 0` to `assert distalSegmentInhibionFactor < 1`.